stroeer/core/v1/article.proto

```diff
@@ -76,3 +89,13 @@ enum AssetType {
   ASSET_TYPE_URL = 3;
   ASSET_TYPE_METADATA = 4;
 }
+
+enum HtmlNodeType {
+  HTML_NODE_TYPE_UNSPECIFIED = 0;
+  HTML_NODE_TYPE_PARAGRAPH = 1;
+  HTML_NODE_TYPE_SUB = 2;
```

Comment:
was ist SUB und was ist SUP?

Reply:
`<sub>` `<sup>` ;)

Reply:
hoch- und tiefstellen (² und ₂)

Reply:
siehe mein kommentar dazu was überhaupt im cms erzeugbar ist. würde nur typen anlegen, die es auch geben kann

Reply:
anhand der ganzen Typen, die du definiert hast, kann ich mir das schwer vorstellen. vielleicht könntest du mal einen echten artikel in dem modell abbilden.

was würde denn aus diesem paragraph in unserem modell werden?

```
    {
       "identifier": "p:2:2.paragraph",
        "id": "doc-1efu1v7oo0",
        "content": {
                "paragraph": "<strong>Lorem</strong> <strong>ipsum</strong> dolor sit amet,<em> ius cu corpora delicata deterruisset</em>. Eam aeque malorum ei, mei <a href=\"http://www.errem.com\">errem</a> alterum ei, mentitum complectitur duo in. Pri et illum contentiones, quaeque praesent te cum, ne rationibus efficiendi sed. Ne duo magna equidem vituperatoribus. Rebum cetero an pro, insolens ⊕ ne ⊖ vel. <br>"
        }
    },
```

Reply:
```id: "$ARTICLE_ID"
body {
  children {
    type: HTML_NODE_TYPE_PARAGRAPH
    children {
      type: HTML_NODE_TYPE_STRONG
      text: "Lorem"
    }
    children {
      type: HTML_NODE_TYPE_TEXT
      text: " "
    }
    children {
      type: HTML_NODE_TYPE_STRONG
      text: "ipsum"
    }
    children {
      type: HTML_NODE_TYPE_TEXT
      text: "dolor sit amet,"
    }
    children {
      type: HTML_NODE_TYPE_EM
      text: " ius cu corpora delicata deterruisset"
    }
    children {
      type: HTML_NODE_TYPE_TEXT
      text: ". Eam aeque malorum ei, mei"
    }
    children {
      type: HTML_NODE_TYPE_ANCHOR
      text: "errem"
      fields {
        key: "href"
        value: "http://www.errem.com"
      }
    }
    children {
      type: HTML_NODE_TYPE_TEXT
      text: " alterum ei, mentitum complectitur duo in.\n  Pri et illum contentiones, quaeque praesent te cum, ne rationibus efficiendi sed.\n  Ne duo magna equidem vituperatoribus. Rebum cetero an pro, insolens \342\212\225 ne \342\212\226 vel. "
    }
  }
}```

Reply:
I like this approach, this way the ui gets more flexibility when it comes to rendering with custom react tags (@LukasBombach might also find this useful), then we need to have a solid parser to generate this tree as fast and render it per request until cached.

Reply:
we use `string` as the type now